index.html

```diff
@@ -1,47 +1,55 @@
 <!DOCTYPE html>
-<html ang="en">
+<html lang="en">
 <head>
     <title>YangZhou</title>
-    <meta charset="ascii"
+    <meta charset="utf-8">
 </head>
 <body>
-<header>
-<nav>
-<ul>
-  <li><a href="/CSD120/index.html">home</a> </li>
-  <li><a href="/CSD120/links.html">Links</a></li>
-  <li><a href="/CSD120/glossary.html">glossary</a></li>
-</ul>
-</nav>
-</header>
-<main>
-<a href="http://node11.codenvy.io:42865//CSD120/jacky.html" target="_blank"><h1>heading1</h1></a>
-<h2>My Three Favorite ball games
-Basketball, football and tennis</h2>
-<ol> <li>Basketball</li> <li>football</li> <li>tennis</li> </ol>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="/CSD120/index.html">home</a> </li>
+                <li><a href="/CSD120/links.html">Links</a></li>
+                <li><a href="/CSD120/glossary.html">glossary</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <h1>heading1</h1>
 
-<p>I am Zhou Yang from China. I like computer ga
-mes and football very much.</p>
-My favorite football club is Manchester City, because their football is very beautiful.
```

Comment:
This paragraph should be wrapped in a `<p>` element.  (Each paragraph should be marked using one `<p>` element